src/node/cli.ts

```diff
@@ -434,7 +426,7 @@ export interface DefaultedArgs extends ConfigArgs {
   "extensions-dir": string
   "user-data-dir": string
   /* Positional arguments. */
-  _: []
+  _: string[] | []
```

Comment:
Haha I never noticed this was just a plain array type.  We could probably leave this at just `string[]` if we wanted!

Reply:
Yeah, it was causing issues telling me it was a tuple and I couldn't access it 😂 I'll remove `[]`